src/kvirc/kernel/KviIrcConnectionServerInfo.cpp

```diff
@@ -275,6 +273,566 @@ KviBasicIrcServerInfo::~KviBasicIrcServerInfo()
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'T': return __tr2qs("T: Prevent CTCPs"); break;
+		case 'V': return __tr2qs("V: Connected over WebTV"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+		case 'a': return __tr2qs("a: Services administrator"); break;
+		case 'd': return __tr2qs("d: Deaf"); break;
+		case 'g': return __tr2qs("g: Can send & read LOCOPS and GLOBOPS"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'p': return __tr2qs("p: Channels hidden from WHOIS"); break;
+		case 'q': return __tr2qs("q: Immune to kicks (except U-Line)"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 't': return __tr2qs("t: Using a vhost"); break;
+		case 'v': return __tr2qs("v: Recipient for infected DCC Send Rejection notices"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+		case 'z': return __tr2qs("z: Connected over SSL"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviInspIRCdIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'c': return __tr2qs("c: Prevent CTCPs"); break;
+		case 'd': return __tr2qs("d: Deaf"); break;
+		case 'g': return __tr2qs("g: Server side ignore"); break;
+		case 'h': return __tr2qs("h: Available for help (HelpOp)"); break;
+		case 'k': return __tr2qs("k: Network service"); break;
+		case 'r': return __tr2qs("r: Registered"); break;
+		case 'x': return __tr2qs("x: Host hidden"); break;
+		case 'B': return __tr2qs("B: Marks as being a bot"); break;
+		case 'G': return __tr2qs("G: Censor bad words"); break;
+		case 'H': return __tr2qs("H: Hide oper status"); break;
+		case 'I': return __tr2qs("I: Channels hidden from WHOIS"); break;
+		case 'Q': return __tr2qs("Q: Marks an IRC Operator as invisible from user lists"); break;
+		case 'R': return __tr2qs("R: Receive private messages from registered nicks only"); break;
+		case 'S': return __tr2qs("S: Strip colors out of private messages"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+	}
+	return KviBasicIrcServerInfo::getUserModeDescription(mode);
+}
+
+const QString & KviCritenIrcServerInfo::getUserModeDescription(QChar mode)
+{
+	switch(mode.unicode())
+	{
+		case 'C': return __tr2qs("C: Can see CHATOPS notices"); break;
+		case 'D': return __tr2qs("D: Has seen DCCALLOW warning message"); break;
+		case 'F': return __tr2qs("F: Recipient for far connect/quit notices"); break;
+		case 'P': return __tr2qs("P: Services administrator"); break;
+		case 'S': return __tr2qs("S: Network service"); break;
+		case 'W': return __tr2qs("W: Recipient for notices on WHOIS"); break;
+		case 'Z': return __tr2qs("Z: Services root administrator"); break;
+		case 'a': return __tr2qs("a: Services operator"); break;
```

Comment:
This is a services class type that has been defined since IRC Services (I'm pretty sure). So Services operator, Services root administrator, Services administrator are correct.